pages/common/gist.md

```diff
@@ -0,0 +1,23 @@
+# gist
+
+> Upload code to https://gist.github.com.
+
+- Upload any number of files:
+
+`gist {{file.txt}} {{file2.txt}}`
+
+- Upload a private file with a description:
+
+`gist -p {{file.txt}} -d {{"A meaningful description"}}`
+
+- Read contents from STDIN with a name:
+
+`{{echo hello world}} | gist -f {{name.rb}}`
+
+- List public and private gists for authed user:
```

Comment:
Shouldn't the user be authenticated for all the commands ? Is there any benefit of saying "authed user" ?